cmd/kubeadm/app/cmd/init.go

```diff
@@ -159,20 +160,23 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 	}
 
 	// adds command flags
-	AddInitConfigFlags(cmd.PersistentFlags(), options.externalcfg, &options.featureGatesString)
-	AddInitOtherFlags(cmd.PersistentFlags(), &options.cfgPath, &options.skipTokenPrint, &options.dryRun, &options.ignorePreflightErrors)
-	options.bto.AddTokenFlag(cmd.PersistentFlags())
-	options.bto.AddTTLFlag(cmd.PersistentFlags())
+	AddInitConfigFlags(cmd.PersistentFlags(), initOptions.externalcfg, &initOptions.featureGatesString)
+	AddInitOtherFlags(cmd.PersistentFlags(), &initOptions.cfgPath, &initOptions.skipTokenPrint, &initOptions.dryRun, &initOptions.ignorePreflightErrors)
+	initOptions.bto.AddTokenFlag(cmd.PersistentFlags())
+	initOptions.bto.AddTTLFlag(cmd.PersistentFlags())
+	options.AddKubeConfigFlag(cmd.Flags(), &kubeConfigFile)
```

Comment:
@fabriziopandini this forces us to have a kubeconfig on the init CLI. :thinking: